src/utils/normalize-otherAttribute-contact.ts

```diff
@@ -0,0 +1,137 @@
+import concat from 'lodash/concat';
+import differenceBy from 'lodash/differenceBy';
+import forEach from 'lodash/forEach';
+import { denormalize } from '../normalize';
+import { ContactInputRequest } from '../normalize/entities';
+
+const supportedContactAttributes = [
+	'firstName',
+	'middleName',
+	'lastName',
+	'fullName',
+	'maidenName',
+	'namePrefix',
+	'nameSuffix',
+	'email',
+	'email2',
+	'workEmail',
+	'workEmail2',
+	'homeEmail',
+	'homeEmail2',
+	'phone',
+	'phone2',
+	'companyPhone',
+	'companyPhone2',
+	'otherPhone',
+	'otherPhone2',
+	'mobilePhone',
+	'mobilePhone2',
+	'homePhone',
+	'homePhone2',
+	'workPhone',
+	'workPhone2',
+	'pager',
+	'pager2',
+	'homeFax',
+	'homeFax2',
+	'workFax',
+	'workFax2',
+	'imAddress',
+	'imAddress1',
+	'imAddress2',
+	'imAddress3',
+	'imAddress4',
+	'imAddress5',
+	'nickname',
+	'homeStreet',
+	'homeCity',
+	'homeState',
+	'homePostalCode',
+	'homeCountry',
+	'homeURL',
+	'workStreet',
+	'workCity',
+	'workState',
+	'workPostalCode',
+	'workCountry',
+	'workURL',
+	'jobTitle',
+	'company',
+	'department',
+	'birthday',
+	'anniversary',
+	'website',
+	'notes',
+	'image',
+	'userCertificate',
+	'assistantPhone',
+	'callbackPhone',
+	'carPhone',
+	'otherCity',
+	'otherCountry',
+	'otherFax',
+	'otherPostalCode',
+	'otherState',
+	'otherStreet',
+	'otherURL',
+	'fileAs',
+	'type'
+];
+export function createContactBody(data: any) {
+	const { attributes, ...rest } = data;
+	const contactAttrs = <Object[]>[];
+
+	forEach(attributes, (val, key) =>
```

Comment:
Any reason we are using `lodash` `foreach` and not `array.foreach` ?

Reply:
The attributes is not an array but an object hence used lodash foreach as it iterates over object as well.